### PR TITLE
Create .gitignore

### PR DESCRIPTION
**Pull request by marcosmarp**
Solves #3

### CODE REVIEW

**Comment by marcosmarp** [comment]
I think that now is working. I can't see the undesired files in the new branch.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Compiled source #
+###################
+*.o
+*.exe
```
Comment (nahueespinosa) [inline .gitignore:4]:
```suggestion
*.exe
LogInDataBase
```

That would be my suggestion.

**Review by nahueespinosa — CHANGES_REQUESTED** [review]
Great news!

Mmh, I don't think `LogInDataBase.dev` is the file you want to exclude as it seems to contain some type of configuration for your project.

I mentioned `LogInDataBase` (without the extension) as that file seems to be the final executable (correct me if I'm wrong). So to remove it you would need to directly write its name in `.gitignore`.

File: .gitignore (hunk 1 above)
Comment (marcosmarp) [reply .gitignore:4]: I just checked and I'm pretty sure that the LogInDataBase and the .dev file were generated by Dev C++. 
The IDE already generated a .exe but i never upload it, I thought (wrongly) that you only uploaded the .exe in the releases, but as I can see now you have to add it to the code too.

The reason why I'm saying those two files are generated by Dev C++ is cause I created a new project in another IDE, copied the .cpp and .o files of this project and compiled and those files weren't generated

Comment (nahueespinosa) [reply .gitignore:4]: Got it, then let's remove all Dev C ++ files from this repository as they depend on the IDE.
The `.exe` would be part of a release. You don't have to add it to the repo, that's why we want git to ignore that file.

**Review by nahueespinosa — APPROVED** [review]
Looking good! Let's merge this 🎉

**Comment by marcosmarp** [comment]
Hey, didn't saw your approval and added the IDE files extensions to the .gitignore file. Is that okay?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+* LogInDataBase
```
Comment (nahueespinosa) [inline .gitignore:5]:
```suggestion
# Dev C++ IDE     #
###################
```
I'd change this. Remove `* LogInDataBase` and separate the following lines specifying the reason why they are being ignored.

I think you can accept the suggestion directly from GitHub and that will create a commit.

**Comment by marcosmarp** [comment]
Ready to go! Merging the PR!

**Pull request merged**